packages/frontend-api/src/Controller/SocialNetworkController.php

```diff
@@ -68,6 +68,7 @@ public function loginAction(Request $request, string $type): Response
                     $loginResultData->showCartMergeInfo,
                     $loginResultData->isRegistration,
                 ),
+                'showCartMergeInfo' => $loginResultData->showCartMergeInfo
```

Comment:
This will not work, it has to be string

```suggestion
                'showCartMergeInfo' => $loginResultData->showCartMergeInfo ? 'true' : 'false',
```

Reply:
Thanks, will fix that